### DISCUSSION TITLE
Mentioning details about  Parameterized Query

### DISCUSSION THREAD

**Discussion by riyafa** [ballerina-platform/ballerina-standard-library#874] [Ideas]
When trying to implement something using databases, I was purely trying to use our examples and ballerina docs to implement my use cases. The doc I feel is not comprehensive enough. There was no specific examples for inserting or updating the database. So when I looked into the Parameterized Query doc, it doesn't talk about what a Parameterized Query is and how it is applicable for Ballerina. Instead it simply assumes, that the seer of this [particular doc](https://ballerina.io/swan-lake/learn/by-example/mysql-parameterized-query.html) is aware of what a Parameterized query is. Or  assumes he/she might  google about it. Wouldn't it be better if we write the doc so that a user who comes to this site does not have to leave it to learn about concepts even though if it is about databases.

**Comment by niveathika** [comment]
The specific doc is now removed and we have improved the bbes so that its more descriptive
